integrated_channels/lms_utils.py

```diff
@@ -67,31 +67,29 @@ def get_single_user_grade(course_id, user):
             'To use this function, this package must be '
             'installed in an Open edX environment.'
         )
-    course_key = CourseKey.from_string(course_id)
-    course_grade = CourseGradeFactory().read(user, course_key=course_key)
+    course_id = CourseKey.from_string(course_key)
+    course_grade = CourseGradeFactory().read(user, course_key=course_id)
```

Comment:
note: every thing that says `course_id` in the grading apis refers to instances of CourseKey which have to be obtained via CourseKey.from_string() from the course_key strings provided from learner_data.py (those enterprise_enrollment.course_id are actually course key strings)